### ISSUE TITLE
No package is available for lua-language-server (3.6.17) for linux arm at the moment.

### ISSUE THREAD

**Issue by xpufx** [williamboman/mason.nvim#1101]
### I've searched open issues for similar requests

Yes

### I've recently downloaded the latest plugin version of mason.nvim

- [ ] Yes

### Problem description

https://github.com/LuaLS/lua-language-server/releases   does not list a linux-arm version.  At https://github.com/LuaLS/lua-language-server/issues/1992 the user is instructed to "Wait for next version" without explanation.

### Expected behavior

I am not sure. Perhaps Mason should fall back to the latest available version?  Perhaps an alternative language server option can be added in general?

### Affected packages

lua-language-server

### Mason output

```Text
[ERROR Wed Mar 15 09:16:21 2023] ...acker/start/mason.nvim/lua/mason-core/installer/init.lua:200: Installation failed for Package(name=lua-language-server) error='...er/start/mason.nvim/lua/mason-core/managers/std/init.lua:52: Failed to download file "https://github.com/LuaLS/lua-language-server/releases/download/3.6.17/lua-language-server-3.6.17-linux-arm64.tar.gz".\nspawn: wget failed with exit code 8 and signal 0.
```

### Installation log

```Text
I don't think this is necessary.
```

### Neovim version (>= 0.7)

0.8.3

### Operating system/version

Linux myhostname 5.15.0-67-generic #74-Ubuntu SMP Wed Feb 22 14:14:39 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux

### Healthcheck

```Text
relevant bits are OK
```

**Comment by williamboman** [comment]
Hey! Yeah I saw this a couple of days ago when [CI failed ](https://github.com/mason-org/mason-registry/pull/808) in the soon-to-be new registry. This new registry fixes issues like these and so it's just a matter of time until I start rolling out generally. If you'd like (and I'd appreciate any early feedback) you could switch to the new registry already today, like so:

```lua
require("mason").setup {
  registries = {
    "github:mason-org/mason-registry",
    "lua:mason-registry.index"
  }
}
```

**Comment by xpufx** [comment]
That went through and installed the language server. (I had to install unzip which I didn't seem to need before. In case you want to mention in documentation)

[INFO  Wed Mar 15 10:03:01 2023] ...acker/start/mason.nvim/lua/mason-core/installer/init.lua:166: Executing installer for Package(name=lua-language-server) version="latest"
[INFO  Wed Mar 15 10:03:02 2023] ...acker/start/mason.nvim/lua/mason-core/installer/init.lua:193: Installation succeeded for Package(name=lua-language-server)

I see the language server loading the workspace and I tried a basic Format which worked.

How does it work anyway? Did it install the previous version?

**Comment by williamboman** [comment]
> That went through and installed the language server. (I had to install unzip which I didn't seem to need before. In case you want to mention in documentation)

`unzip` is one of the core requirements already mentioned in the docs. This in particular seems to be a common issue, I'm personally really surprised that people don't have `unzip` installed already, maybe I spend too much time in the terminal 😂 

`lua-language-server` should not require `unzip` however, so that sounds really weird. If you'd like could you re-install the package with `DEBUG` logging enabled (`:h mason-debugging`)? (remember to reset the log level after)

> How does it work anyway? Did it install the previous version?

The "latest" version is encoded in the [package definition itself](https://github.com/mason-org/mason-registry/blob/main/packages/lua-language-server/package.yaml#L13), so it resolves which version to install entirely "offline".

**Comment by xpufx** [comment]
You're right unzip should be installed. This is a new vm where I copied over my neovim config including existing directories. I also had to install wget. This is not an issue at all. Just pointed it out in case it is somehow a new requirement.

As for where unzip is used, it's not lua-language-server but mason-registry itself. 

3:[ERROR Wed Mar 15 10:02:11 2023] ...r/start/mason.nvim/lua/mason-registry/sources/github.lua:178: Failed to install registry GitHubRegistrySource(repo=mason-org/mason-registry). "Failed to unpack registry contents: unzip is not executable"
